pnparser/templates/pai.py

```diff
@@ -0,0 +1,16 @@
+from typing import TYPE_CHECKING
+from ..helpers import Helper
```

Comment:
can you use absolute paths for all the imports like `from patchnotesparser.helpers import Helper

btw Pycharm will do this automatically for you if you press Alt+Enter on something you're importing

Reply:
I am using Pylance and for some reason it doesn't like fully qualified imports, it cannot resolve them, I'll figure out why later, but since the change works with Discord I'll just do it for now.

Reply:
You should definitely 100% switch to Pycharm, it's free and 100000000% more powerful than VSCode+addons. The amount of auto-refactors & project scans it's able to do is unreal, even if in the free version. I've never recommended an IDE this strongly, including SublimeText lol. Their hotkeys totally suck initially but you can rebind them and/or import VSCode defaults, though the "import XYZ editor defaults" isn't super robust. Definitely give it at least a week chance!!

Reply:
Thanks, I will!